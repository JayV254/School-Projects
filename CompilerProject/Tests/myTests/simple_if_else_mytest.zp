int main() {
        int a;
        a = 3;

        if (a < 20) {
                puts("a is less than 20");
        }
        else {
                puts("a is greater than 20");
        }

        return 0;
}
