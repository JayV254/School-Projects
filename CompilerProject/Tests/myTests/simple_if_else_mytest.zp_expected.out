a is less than 20
